csunplugged/topics/content/en/programming-challenges-structure.yaml

```diff
@@ -1,9 +1,11 @@
 languages:
     scratch:
         name: Scratch
+        order: 1
```

Comment:
we use `number` everywhere else (lessons, curric integrations), why is it `order` here?